src/components/base-address/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bcrs-shared-components/base-address",
-  "version": "2.0.45",
+  "version": "2.0.46",
```

Comment:
Revert this. Step 14 in the README (which I or another admin will do) will take care of this.